unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/Settings.cs

```diff
@@ -89,14 +104,34 @@ private void LoadGeneralSettings()
             }
         }
 
+        private void LoadAudioSettings() {
```

Comment:
```suggestion
        private void LoadAudioSettings() 
        {
```
Please put line breaks before the braces to conform to our general style in the codebase.